.github/workflows/build.yaml

```diff
@@ -0,0 +1,61 @@
+name: Build
+on:
+    push:
+    pull_request:
+    workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        type: [tar.xz, qcow2]
+        arch: [amd64, arm64]
+    env:
+      GITHUB_ACTION: true
```

Comment:
This is clobbering the default environment variable (there's no need to set this).
https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables